### PR TITLE
Improve gradle.md

### PR DESCRIPTION
**Pull request by ham1**

### CODE REVIEW

File: gradle.md [hunk 1]
```diff
@@ -47,52 +50,79 @@ Technically speaking, `clean` should not be required (every time it is required
...
+      gw checkstyleAll
+      
+      # Run Spotless checks
+      gw spotlessCheck
+      
+      # Fix any issues found by Spotless
+      gw spotlessApply
+      
+      # Run spotlessApply and checkstyleAll
+      gw style
```
Comment (vlsi) [inline gradle.md]: Could you please move `gw style` and `gw checkstyle` commands to the first?
The rest commands are lowlevel, and I don't expect people would need them.

File: gradle.md [hunk 2]
```diff
@@ -113,25 +143,30 @@ Gradle automatically tracks task dependencies, so if you modify a file in `/src/
...
+> performed with `gw :src:dist:assemble`
 
 ## Releasing
 
       # Builds the project, pushes artifacts to svn://.../dev, stages artifacts to Nexus staging repository
       gw prepareVote -Prc=1 -Pasf
 
-> **Note:** The above step uses [an asf-like release environment](https://github.com/vlsi/asflike-release-environment), so it does not alter public repositories
+> **Note:** The above step uses [an asf-like release environment](https://github.com/vlsi/asflike-release-environment),
+> so it does not alter public repositories
```
Comment (vlsi) [inline gradle.md]: Ah. That is a bug `-Pasf` means "publish to ASF servers"

Comment (ham1) [reply gradle.md]: I'm not sure what you mean,  would you be able to merge the PR and then correct this section yourself? Also I noticed it mentions SVN, I presume this is no longer true?

Comment (vlsi) [reply gradle.md]: SVN is used for all Apache projects for distributing the releases via https://dist.apache.org/repos/dist

File: gradle.md [hunk 3]
```diff
@@ -47,52 +50,81 @@ Technically speaking, `clean` should not be required (every time it is required
...
       # Run RAT
       gw rat
+      
+### Code Formatting
+      
+      # Run spotlessApply and checkstyleAll
+      gw style
+      
+      # Run checkstlye for all
+      gw checkstyleAll
```
Comment (vlsi) [inline gradle.md]: This should be `gw checkstyle`, not `checkstyleAll`

**Pull request merged**